sites/public/lib/filterUrlLink.tsx

```diff
@@ -0,0 +1,17 @@
+import { encodeToFrontendFilterString, ListingFilterState } from "@bloom-housing/ui-components"
+
+export function getFilterUrlLink(eligibilityRequirements) {
+  /** This is where any logic for the filtered results from the eligibility questionnaire should be located.*/
+
+  const SENIOR_AGE = 62
+
+  const params: ListingFilterState = {}
+
+  if (eligibilityRequirements.age < SENIOR_AGE) {
+    params.seniorHousing = false
+  } else {
+    params.seniorHousing = true
+  }
```

Comment:
Remove this true one; since it's for eligibility, so we want to show everything they're eligible for, which includes housing not specific to this community.

Reply:
removed